src/components/pages/Login.vue

```diff
@@ -32,4 +39,9 @@ h1 {
   margin-top: 1%;
 }
 
+span {
```

Comment:
Add COLORS.TEXT_PRIMARY and FONT_SIZES.IMPORTANT

Reply:
Font is now calculated, when full width it has IMPORTANT size